src/com/google/common/css/compiler/gssfunctions/ColorUtil.java

```diff
@@ -43,6 +45,39 @@ public static Color hsbToColor(float[] inputHsb) {
     return Color.getHSBColor(inputHsb[H], inputHsb[S], inputHsb[B]);
   }
 
+  public static float[] hsbToHsl(float[] inputHsb) {
```

Comment:
Since the class is package-private (default visibility), it doesn't really make sense to make the methods be public. Can you remove the public from your methods and the existing ones?

And can you add some JavaDoc?